bin/pbiviz.js

```diff
@@ -61,6 +67,148 @@ if (program.args.length > 0) {
     }
 }
 
+function removeCertFiles(certPath, keyPath, pfxPath) {
+    try {
+        fs.unlinkSync(certPath);
+    } catch (e) {
+        if (!e.message.indexOf("no such file or directory")) {
```

Comment:
does it specify which path is missing? certPath?

Reply:
We need to remove old cert before generating the new one.
The tool checks the exception message to suppress exceptions if the file wasn't found.
Because in the first installation the tool doesn't have any generated certs. In this case, we can get an error that file doesn't exist.

And concrete path doesn't matter.